modules/teadsBidAdapter.js

```diff
@@ -221,4 +234,40 @@ function _validateId(id) {
   return (parseInt(id) > 0);
 }
 
+/**
+ * Get FLoC parameters to be sent in the bid request.
+ * @param `{id: string, version: string} | undefined` optionalFlocId FLoC user ID object available if "flocIdSystem" module is enabled.
+ * @returns `{} | {cohortId: string} | {cohortVersion: string} | {cohortId: string, cohortVersion: string}`
+ */
+function getFLoCParameters(optionalFlocId) {
+  if (!optionalFlocId) {
+    return {};
+  }
+  const cohortId = optionalFlocId.id ? { cohortId: optionalFlocId.id } : {};
+  const cohortVersion = optionalFlocId.version ? { cohortVersion: optionalFlocId.version } : {};
+  return { ...cohortId, ...cohortVersion };
+}
+
+/**
+ * Get unified ID v2 parameter to be sent in bid request.
+ * @param `{id: string} | undefined` optionalUid2 uid2 user ID object available if "uid2IdSystem" module is enabled.
+ * @returns `{} | {unifiedId2: string}`
+ */
+function getUnifiedId2Parameter(optionalUid2) {
+  return optionalUid2 ? { unifiedId2: optionalUid2.id } : {};
+}
+
+/**
+ * Get the first-party cookie Teads ID parameter to be sent in bid request.
+ * @returns `{} | {firstPartyCookieTeadsId: string}`
+ */
+function getFirstPartyTeadsIdParameter() {
+  const storage = getStorageManager(GVL_ID, BIDDER_CODE);
```

Comment:
Think it would be better to just init your storage manager top level of module

You are calling this function every single time anyways.